src/sentry/integrations/discord/requests/base.py

```diff
@@ -116,8 +116,20 @@ def authorize(self) -> None:
         timestamp: str | None = self.request.META.get("HTTP_X_SIGNATURE_TIMESTAMP")
         body: str = self.request.body.decode("utf-8")
 
+        logger.info(
+            "discord.authorize.request",
+            extra={
+                "public_key": public_key,
+                "signature": signature,
+                "timestamp": timestamp,
+                "body": body,
+            },
+        )
+
         if signature and timestamp and verify_signature(public_key, signature, timestamp + body):
             return
+        else:
+            logger.info("discord.verify.signature.failure")
```

Comment:
@17hogeju we should pass the same variables for `extra` here. Also, probably should be a warning.